### PR TITLE
Show issue in breadcrumb when parent_id is not in the path

### LINKED ISSUE #2342: Coin: show Issue breadcrumbs even if not using parent path
When I go to a coin at the `/catalog/[id]`, it should show the Numismatic Issue parent in the breadcrumbs.

Child of #2025

### PR DESCRIPTION
**Pull request by christinach**
closes #2342

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -132,9 +132,17 @@ def set_parent_document
       query = "member_ids_ssim:id-#{params['id']}"
       _, result = search_results(q: query, rows: 1)
       @parent_document = result.first if result.first
+    elsif params[:parent_id].nil? && @document.decorated_resource.is_a?(CoinDecorator)
```
Comment (tpendragon) [inline app/controllers/catalog_controller.rb]: This seems like a good feature to do for everything. Maybe change it to `@document.decorated_resource.try(:decorated_parent)`

File: app/controllers/catalog_controller.rb [hunk 2]
```diff
@@ -132,9 +132,17 @@ def set_parent_document
...
+      set_coin_parent
     end
   end
 
+  def set_coin_parent
+    return unless CoinDecorator.new(resource).decorated_parent
```
Comment (tpendragon) [inline app/controllers/catalog_controller.rb]: You can get rid of this if you check for it above.

File: spec/resources/coins/coin_feature_spec.rb [hunk 3]
```diff
@@ -1,16 +1,21 @@
 # frozen_string_literal: true
 require "rails_helper"
 
-RSpec.feature "Coins" do
+RSpec.feature "Coins", js: true do
```
[the remark this answers is not in the document]
Comment (christinach) [reply spec/resources/coins/coin_feature_spec.rb]: you're right. it doesn't. I think I tested in the beginning a style with selenium and added it. removed

**Comment by christinach** [comment]
@tpendragon thanks

**Comment by christinach** [comment]
I'm checking a local error with a 'boxless ephemera project'.

**Comment by christinach** [comment]
ok, it looks like I had to rebase.

File: app/controllers/catalog_controller.rb [hunk 4]
```diff
@@ -132,9 +132,16 @@ def set_parent_document
       query = "member_ids_ssim:id-#{params['id']}"
       _, result = search_results(q: query, rows: 1)
       @parent_document = result.first if result.first
+    elsif params[:parent_id].nil? && @document.decorated_resource.try(:decorated_parent)
+      set_coin_parent
     end
   end
 
+  def set_coin_parent
+    params[:parent_id] = @document.decorated_resource.decorated_parent["id"].id
```
Comment (tpendragon) [inline app/controllers/catalog_controller.rb]: Oh, instead of `.id` I'd do `.to_s`.

Comment (christinach) [reply app/controllers/catalog_controller.rb]: thanks @tpendragon

**Pull request merged**